pkg/apis/kops/cluster.go

```diff
@@ -132,7 +132,13 @@ type ClusterSpec struct {
 	//   missing: default policy (currently OS security upgrades that do not require a reboot)
 	UpdatePolicy *string `json:"updatePolicy,omitempty"`
 
+	// Use an existing custom cloud security policy for the instances.  One example is to specify the name
+	// of an AWS IAM role for the master and another for the nodes.
+	// Map is keyed by: master, node
+	CustomPolicies *map[string]string `json:"customPolicies,omitempty"`
```

Comment:
No reason to have a pointer to a map, I believe; just use a map.

And should we just have a strongly typed object i.e.

```
iam:
  master: override-master
  node: override-node
```